src/Domain/Creators/Sphere.cpp

```diff
@@ -227,7 +227,8 @@ Sphere::Sphere(
       which_wedges_ != ShellWedges::All) {
     PARSE_ERROR(
         context,
-        "Can only apply boundary conditions when using the full shell. "
+        "Can only apply boundary conditions when the outer boundary of the "
+        "domain is a full sphere. "
```

Comment:
Formatting?

Reply:
I sometimes leave line breaks like this to keep git diffs small when editing these messages. Reflowing the full paragraph creates big diffs and can be annoying, and line breaks avoid that. Just a minor thing, but it helps sometimes.